doc/whats-new.rst

```diff
@@ -34,6 +34,8 @@ New Features
 Bug fixes
 ~~~~~~~~~
 
+- Fix bug where reference times without padded years (e.g. "since 1-1-1") would lose their units when being passed by :py:func:`encode_cf_datetime` (:pull:`4506`).
```

Comment:
This actually fixes #4422 (perhaps you were already aware!).
```suggestion
- Fix bug where reference times without padded years (e.g. "since 1-1-1") would lose their units when being passed by :py:func:`encode_cf_datetime` (:issue:`4422`, :pull:`4506`).
```
Should we also make it clearer that before this PR, without `cftime`, we used to silently parse these reference dates incorrectly, at least based on the [CF conventions](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.8/cf-conventions.html#time-coordinate)?

Reply:
> This actually fixes #4422 (perhaps you were already aware!).

I actually hadn't seen that issue, thanks for linking

> Should we also make it clearer that before this PR, without `cftime`, we used to silently parse these reference dates incorrectly, at least based on the [CF conventions](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.8/cf-conventions.html#time-coordinate)?

+1 from me, will wait for opinion of others before changing.

Reply:
> Should we also make it clearer that before this PR, without cftime, we used to silently parse these reference dates incorrectly, at least based on the CF conventions?

:+1:

Reply:
I've cleaned up the whatsnew entry now, see what you think